backend/src/routes/api/storage-class/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { FastifyRequest } from 'fastify';
+import { K8sResourceCommon, KubeFastifyInstance, StorageClassConfig } from '../../../types';
+import { isHttpError } from '../../../utils';
+import { errorHandler } from '../../../utils';
+
+export async function updateStorageClassMetadata(
+  fastify: KubeFastifyInstance,
+  request: FastifyRequest<{ Body: StorageClassConfig }>,
+): Promise<{ success: boolean; error: string }> {
```

Comment:
I think `{ success: boolean; error: string }` is a type we have already called `ResponseStatus` in `src/types`

Reply:
that is a frontend type. i can add it to the backend too